hail/src/main/scala/is/hail/backend/local/LocalBackend.scala

```diff
@@ -2,37 +2,50 @@ package is.hail.backend.local
 
 import is.hail.expr.JSONAnnotationImpex
 import is.hail.expr.ir._
-import org.json4s.jackson.JsonMethods
+import is.hail.utils._
+import is.hail.utils.{ExecutionTimer, Timings}
+import org.json4s.DefaultFormats
+import org.json4s.jackson.{JsonMethods, Serialization}
 
 object LocalBackend {
   def executeJSON(ir: IR): String = {
     val t = ir.typ
-    val value = execute(ir)
-    JsonMethods.compact(
-      JSONAnnotationImpex.exportAnnotation(value, t))
+    val (value, timings) = execute(ir)
+    val jsonValue = JsonMethods.compact(JSONAnnotationImpex.exportAnnotation(value, t))
+    log.info(s"Execution times: ${ timings.value }")
```

Comment:
this isn't going to be particularly readable -- maybe add a function on `Timings` for logging that prints the readable times with one `log.info` call per key?

Reply:
Good point, printed the readable time, one stage at a time